### ISSUE TITLE
Missing calibration code

### ISSUE THREAD

**Issue by zhimin-wang** [grebtsew/Tobii_Glasses_2_Python3_And_C#2]
Hello, calibration functions aren't called in your code. And the python code can be converted to c++ code easily? C++ code is more convenient because of my needs.

**Comment by grebtsew** [comment]
Hi,

No, in this implementation I currently only show how to, in the easiest way, receive data from Tobii Glasses 2. I have code for more usages of the REST api, such as creating new project, participants and calibrations. Do you want me to update the implementation with these features? 

In this implementation we communicate with the Tobii Glasses 2 REST api. A REST api is a web service and works fine with almost every language ​​that support internet protocols. Basicly what I'm trying to say is that all you need to do is to send some HTTP requests towards your Tobii device. Check the Tobii Glasses 2 documentation for more information about which address and what the request should contain.  [Link to "Send HTTP REST Request in c++"](https://www.codeproject.com/Articles/1244632/Making-HTTP-REST-Request-in-Cplusplus)

If you need me to I could make a C++ example, however I do not currently have any Tobii Glasses 2 to test against. 

Cheers!

**Comment by grebtsew** [comment]
Worth noteing, I receive video streams via RTSP with the library OpenCV which is written in C.

**Comment by zhimin-wang** [comment]
Thank you for your reply. 
I try to receive video streams via RTSP in C++ as follow:
```
string s = "rtsp://192.168.71.50:8554/live/scene";
cv::VideoCapture vcap;
if(!vcap.open(s)) 
{
        std::cout << "Error opening video stream or file" << std::endl;
        return;
 }
while (1)
{
        Mat frame;
	vcap.read(frame);
        ……
}
```
But I am not familiar with the HTTP protocol in C++, and I can't see the link you sent due to network restrictions. Can you help me implement a C++ example？You don't have to worry about whether it can run successfully because I can test it if it works.

Thank you very much again.

**Comment by grebtsew** [comment]
I can absolutely create a C++ example, I'll let you know when I'm done.

**Comment by zhimin-wang** [comment]
Thank you for your sincere help.

**Comment by grebtsew** [comment]
Hi again,

I have now added the calibration code in the python implementation.  

I also added a simple C++ implementation (Features include: display rtsp streams, udp sockets data stream, rest api functions) that probably works. Let me know if you have any problems or if you have any questions. I had some struggles installing the needed packages so I added little information about that in the readme file.

Cheers!
